recipe/install.bat

```diff
@@ -4,6 +4,8 @@ cmake --install . --prefix "%LIBRARY_PREFIX%"
 if errorlevel 1 exit 1
 
 if [%PKG_NAME%] == [libjxl] (
+  del "%LIBRARY_LIB%\jxl_extras_codec.lib"
```

Comment:
isn/t this needed to  link to the dll?

Reply:
It links with a ton of media libraries like `libpng`, `openexr`, etc.[^1] Tool binaries like `cjxl` depend on those libraries to convert images between jpeg-xl and other formats, but the core `libjxl` library doesn't, which is why I created a separate `libjxl-tools` to handle these dependencies. For the same reason, I think this library should be put into `libjxl-tools`, despite that it's a library and not a tool.

[^1]: https://github.com/libjxl/libjxl/blob/8f22cb1fb98ed27ceee59887bd291ef4d277c89d/lib/jxl_extras.cmake#L158

Reply:
can you add this as a comment in the sourc,e it would help in the future for people not "undoing" your work.

Reply:
sometimes i build once without all the dependnecies, then again with them.

this ones does so with the graphics dependenices:
https://github.com/conda-forge/hugin-feedstock/blob/main/recipe/meta.yaml

Reply:
> can you add this as a comment in the sourc,e it would help in the future for people not "undoing" your work.

Sure.

> sometimes i build once without all the dependnecies, then again with them.
> 
> this ones does so with the graphics dependenices: https://github.com/conda-forge/hugin-feedstock/blob/main/recipe/meta.yaml

Building twice with cache is ideal but I suspect I didn't do this for good reasons. Changing the configure parameters caused a full rebuild, linking issues, or something like that. We could give it another try in a separate PR.